### ISSUE TITLE
[Bug]在torch1.8 ,cuda试运行知识蒸馏模型时出现问题

### ISSUE THREAD

**Issue by worker-ljw** [open-mmlab/mmrazor#355]
### Describe the bug

在选择配置文件为：cwd_logits_pspnet_r101-d8_pspnet_r18-d8_4xb2-80k_cityscapes-512x1024.py进行模型蒸馏时（其他位置未作改变），在代码运行至    runner = Runner.from_cfg(cfg)处，后经debug至from_cfg函数内发现代码会在run = cls(...)位置卡顿，等待半小时后才开始运行后面的代码。
![image](https://user-images.githubusercontent.com/105470801/203334756-b8da6258-7464-441c-a4d7-417c2493d1a7.png)

\[here\]

### To Reproduce

The command you executed.

```shell
[here]
```

### Post related information

1. The output of `pip list | grep "mmcv\|mmrazor\|^torch"`
   \[here\]
2. Your config file if you modified it or created a new one.

```python
[here]
```

3. Your train log file if you meet the problem during training.
   \[here\]
4. Other code you modified in the `mmrazor` folder.
   \[here\]

### Additional context

\[here\]

**Comment by wilxy** [comment]
Thank you for your issue.

`runner` builds `dataloader`, `model` and so on.
If you pause for a long time after the terminal displays like:
```http loads checkpoint from path: XXXXXXX```
the biggest possibility is that time is spent on downloading the checkpoint when building model.

The model has been downloaded normally if there is no longer a long time consumption in the subsequent distillation.
If this situation still exists, please put your config and terminal display in this issue.

**Comment by worker-ljw** [comment]
This is the config and terminal display:
torch1.8 
cuda10.2
![image](https://user-images.githubusercontent.com/105470801/203454250-1f98e9b7-ac27-47d7-ac39-52f1c03f884b.png)

![1669167530271](https://user-images.githubusercontent.com/105470801/203453969-218484da-1111-4a51-a9cf-9844ba9148c1.jpg)

**Comment by wilxy** [comment]
The result of my local operation is somewhat different from yours, and the loading time after checkpoint download is very short.
Yours:
![image](https://user-images.githubusercontent.com/43191879/203463164-8f9479e7-a3ab-45e3-bcd9-e633f0d235e5.png)
Mine:
![image](https://user-images.githubusercontent.com/43191879/203463231-fc6a6b03-3a82-427f-a1b3-23934d6fc645.png)

The original config of `cwd_logits_pspnet_r101-d8_pspnet_r18-d8_4xb2-80k_cityscapes-512x1024.py` is
```
model = dict(
    _scope_='mmrazor',
    type='SingleTeacherDistill',
    architecture=dict(cfg_path=student_cfg_path, pretrained=False),
    teacher=dict(cfg_path=teacher_cfg_path, pretrained=False),
    teacher_ckpt=teacher_ckpt,
    ...
)
```
with the **`SingleTeacherDistill`**  type of the model.
There may be some problems in your config, please put the complete config in this issue.

**Comment by worker-ljw** [comment]
This is the complete config.But i guess it could be the GPU(NVIDIA 3060).When i select a pruning config, the same problem has arisen.
![微信图片_20221123142139](https://user-images.githubusercontent.com/105470801/203484535-16c46a9f-a7c4-496c-8941-43db34d8e819.png)

![微信图片_202211231421391](https://user-images.githubusercontent.com/105470801/203484550-cbb2300d-ac5b-4db5-89a1-bfa70a6b6979.png)

![微信图片_202211231421392](https://user-images.githubusercontent.com/105470801/203484562-71303c3e-a3e2-4c44-85c6-cc460ab462ad.png)

**Comment by worker-ljw** [comment]
I wish these extral information can be helpful for  you(the loss is nan).
![99b2b884e9f3e3b1bbc94098cd197e9](https://user-images.githubusercontent.com/105470801/203486020-9008c45b-4b9d-4b23-be9f-861aab422937.png)
![image](https://user-images.githubusercontent.com/105470801/203486047-b3b60e11-2a68-4340-be26-a727805621de.png)

**Comment by wilxy** [comment]
I use the same config as you, the `loss` has not become `nan`.
The same problem occurs in a `pruning `algorithm, so we suggest you train a simple classification network like `VGG` (or a segmentation network) and not use `MMRazor` to simplify the problem.
Please let me know if the `loss` is normal in the above case.

**Comment by wilxy** [comment]
> This is the complete config.But i guess it could be the GPU(NVIDIA 3060).When i select a pruning config, the same problem has arisen.
>
Additional, I did the above verifications using the NVIDIA A100, which has the same architecture (Ampere) as your NVIDIA RTX 3060. Basically, the influence of GPU can be ruled out.

**Comment by worker-ljw** [comment]
I have created a new env(cuda 10.2 torch1.8), then this problem still exists.
Meanwhile, I have created another new env and just updated cuda and torch(cuda 10.2 -> cuda11.3 torch 1.8 -> torch11.2) ,then this problem be solved.

**Comment by wilxy** [comment]
I'm glad to hear this issue has been resolved. 
According to your latest description, it should be caused by incompatible versions of `pytorch` and `cuda`.
I will close this issue. Please reopen it if there are any problems in the future.
